### ISSUE TITLE
False positive: Expect has unknown modifier (vitest/valid-expect)

### ISSUE THREAD

**Issue by alexkiro** [vitest-dev/eslint-plugin-vitest#507]
It seems this is wrongly marked as incorect:

```js
  expect("foo bar").to.include("foo");
```

The rule is incorrectly triggered, as this is in fact valid :
```
  6:3   error  Expect has unknown modifier                 vitest/valid-expect
```

**Comment by AllanOricil** [comment]
I have the same issue for `expect().to.have.property()`

Tests pass without any issues. However, this plugin doesnt like it
<img width="822" alt="image" src="https://github.com/user-attachments/assets/331caa3b-cd5e-4586-91fa-e554d70f5ca9">
